src/metax_api/tests/api/rpc/base/views/file_rpc.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of the Metax API service
+#
+# Copyright 2017-2018 Ministry of Education and Culture, Finland
+#
+# :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
+# :license: MIT
+
+from django.core.management import call_command
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from metax_api.models import CatalogRecord, Directory, File
+from metax_api.tests.utils import TestClassUtils, test_data_file_path
+
+
+class FileRPCTests(APITestCase, TestClassUtils):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Seems like setUpClass() has been defined here non-necessarily, since it only calls super() ? (entirely possible you copied this from somewhere in the code base, where it was like that, but the original use of this would also be pointless)